homepage/homepage/components/nav.tsx

```diff
@@ -14,6 +14,10 @@ export function JazzNav() {
                     title: "Docs",
                     href: "/docs",
                 },
+                {
+                    title: "Showcase",
```

Comment:
```suggestion
                    title: "Built With Jazz",
```

Reply:
Not sure about this. Looks kinda weird?

<img width="1258" alt="image" src="https://github.com/user-attachments/assets/9a67d8fc-7c18-4957-b935-6b8f20b10dbb">

Reply:
Ok nvm it's more obvious this way, but I would write it sentence case: "Built with Jazz"